client/src/App.js

```diff
@@ -22,7 +23,10 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <div>
+          <Route path="/" exact component={MovieList}></Route>
```

Comment:
good job with setting up your route paths and directing them to the right components